frontend/public/components/dashboards-page/with-dashboard-resources.tsx

```diff
@@ -63,7 +63,7 @@ export const withDashboardResources = <P extends DashboardItemProps>(WrappedComp
         const queryResultChanged = this.queries.some(query =>
           this.props[RESULTS_TYPE.PROMETHEUS].getIn([query, 'result']) !== nextProps[RESULTS_TYPE.PROMETHEUS].getIn([query, 'result'])
         );
-        const alertsResultChanged = this.props[RESULTS_TYPE.ALERTS].getIn([ALERTS_KEY, 'result']) !== nextProps[RESULTS_TYPE.PROMETHEUS].getIn([ALERTS_KEY, 'result']);
+        const alertsResultChanged = this.props[RESULTS_TYPE.ALERTS].getIn([ALERTS_KEY, 'result']) !== nextProps[RESULTS_TYPE.ALERTS].getIn([ALERTS_KEY, 'result']);
```

Comment:
this change does not look like the change that was made in 4.3/master:
https://github.com/openshift/console/pull/2805/files#diff-e9d97e81eb04b162aba6e3886dbd6d51R70

in particular master is still using `nextProps[RESULTS_TYPE.PROMETHEUS]` not `nextProps[RESULTS_TYPE.ALERTS]`

can you confirm this change is correct?

Reply:
we compare old/new URL, PROMETHEUS and ALERTS in master/4.3 https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/with-dashboard-resources.tsx#L81-L97

and in this PR we do the same  https://github.com/openshift/console/blob/423d2a399f21bad3f711ebfb0e657c1064196560/frontend/public/components/dashboards-page/with-dashboard-resources.tsx#L60-L66

Reply:
ok, well that change was done in this PR:
https://github.com/openshift/console/pull/2806

which is not what https://bugzilla.redhat.com/show_bug.cgi?id=1781053 references (https://github.com/openshift/console/pull/3638)

it would be good to clean up the BZ references so we can track the correct set of changes that were required in each z-stream.